src/components/cards/BiobankReportCard.vue

```diff
@@ -119,13 +166,17 @@
     name: 'biobank-report-card',
     data () {
       return {
-        collapsed: true
+        collapsed: true,
+        quality_logo: {height: 4, width: 9}
```

Comment:
If these values are fixed anyways, I'd put it in the style section

Reply:
I did this, but the styling was somehow not applied when I did.